backend/app/main.py

```diff
@@ -204,6 +212,58 @@ def get_summary(appid: str, response: Response):
     return None
 
 
+@app.post("/generate-download-token", status_code=200)
+def get_download_token(appids: List[str], login=Depends(logins.login_state)):
+    """Generates a download token for the given app IDs."""
+
+    if not login["state"].logged_in():
+        return JSONResponse({"detail": "not_logged_in"}, status_code=401)
+    user = login["user"]
+
+    def canon_app_id(app_id: str):
+        """
+        For .Locale, .Debug, etc. refs, we only check the base app ID. However, when we generate the token, we still
+        need to include the suffixed version.
+        """
+        auto_suffixes = ["Locale", "Debug"]
```

Comment:
Can't find any examples, but I'm wondering, if these can end up being in lowercase? Then this logic would fail.

Reply:
They can't be as far as I know